sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/StringUtils.scala

```diff
@@ -27,21 +28,28 @@ object StringUtils {
   // replace the % with .*, match 0 or more times with any character
   def escapeLikeRegex(v: String): String = {
```

Comment:
nit: while you are at it, can you also make the var names better ?

`v` -> `input`
`c` -> `currentChar`
`prev` -> `previousChar`